fvm/blueprints/system.go

```diff
@@ -25,6 +25,13 @@ func SystemChunkTransaction(chain flow.Chain) (*flow.TransactionBody, error) {
 		return nil, fmt.Errorf("could not get system contracts for chain: %w", err)
 	}
 
+	// Temporary workaround because the heartbeat resources need to be moved
+	// to the service account.
+	// Remove after v0.31
```

Comment:
```suggestion
	// Temporary workaround because the heartbeat resources need to be moved to the service account:
	//  - the system chunk will attempt to load both Epoch and VersionBeacon resources from either the service account or the staking account
	//  - the service account committee can then safely move the resources at any time
	//  - once the resources are moved, this workaround should be removed (after version v0.31)
```